src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -252,8 +252,10 @@ int main(int argc, char **argv)
 	}
 
 	env = getenv("WITH_VALGRIND");
```

Comment:
If this exe is running under valgrind, we can use the macro D_ON_VALGRIND rather than relying on the envirable to be set

Reply:
Do we have a special build for run under Valgrind? I thought we use the same binaries from regular build and just run them under Valgrind.

Reply:
No special build but there's a function/macro you can call which will tell you if valgrind is used or not, so you can adjust tunables appropriately.

Reply:
Thanks.